CHANGELOG.md

```diff
@@ -101,6 +101,7 @@
 
 * [7923](https://github.com/grafana/loki/pull/7923) **manohar-koukuntla**: Add zone aware ingesters in jsonnet deployment
 * [8855](https://github.com/grafana/loki/pull/8855) **JoaoBraveCoding**: Add gRPC port to loki compactor mixin
+* [8880](https://github.com/grafana/loki/pull/8880) **JoaoBraveCoding**: Normalize headless service name for query-frontend/scheduler
```

Comment:
```suggestion
* [8880](https://github.com/grafana/loki/pull/8880) **JoaoBraveCoding**: Mixins: normalize headless service name for query-frontend/scheduler
```

Reply:
Since this is already under `#### Jsonnet` should I really add the prefix?

Reply:
Didn't realise that, disregard 👍